scripts/common_startup.sh

```diff
@@ -121,12 +120,12 @@ if [ $SET_VENV -eq 1 -a $CREATE_VENV -eq 1 ]; then
                     echo "Creating Conda environment for Galaxy: $GALAXY_CONDA_ENV"
                     echo "To avoid this, use the --no-create-venv flag or set \$GALAXY_CONDA_ENV to an"
                     echo "existing environment before starting Galaxy."
-                    $CONDA_EXE create --yes --name "$GALAXY_CONDA_ENV" 'python=2.7' 'pip>=9' 'virtualenv>=16' -c 'conda-forge'
-                    source activate "$GALAXY_CONDA_ENV"
-                    virtualenv "$GALAXY_VIRTUAL_ENV"
+                    $CONDA_EXE create --yes --override-channels --channel conda-forge --channel defaults --name "$GALAXY_CONDA_ENV" 'python=2.7' 'pip>=9' 'virtualenv>=16'
                     unset __CONDA_INFO
                 fi
+                conda_activate
```

Comment:
We're not activating the `_galaxy_` environment here, so the `virtualenv` is not the special virtualenv installed into `$GALAXY_CONDA_ENV`

Reply:
Not sure I understand your comment, we are activating the `$GALAXY_CONDA_ENV` conda environment, which by default is `_galaxy_` (line 117).

Reply:
Where is that happening ?

Reply:
Ah, OK, conda_activate is not what conda ships, it's a custom function.  OK, I see.

Reply:
Yes, in `scripts/common_startup_functions.sh`